### PR TITLE
create comments on release

### PR DESCRIPTION
**Pull request by coryschwartz**
This submits a bunch of testground tests and then reports them back to you in a comment.

To see an example of what this looks like, you can see https://github.com/coryschwartz/lotus/issues/20

### CODE REVIEW

**Comment by magik6k** [comment]
We already have https://github.com/filecoin-project/lotus/blob/master/.circleci/config.yml#L302, would it maybe makes sense to trigger this in circle too?

**Comment by coryschwartz** [comment]
@magik6k The benefit this has over CircleCI is just the comment it makes, so you can see the result of the tests in in the issue. It's just to make it easier when following the release checklist to see that tests are passing.

**Comment by jennijuju** [comment]
@magik6k could you please take a look at this when your time permits?

File: .github/workflows/testground-release-issue.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+            plan_directory: testplans/lotus-soup
+            composition_file: testplans/lotus-soup/_compositions/baseline-k8s-3-3.toml
+          - backend_addr: ci.testground.ipfs.team
+            backend_proto: https
+            plan_directory: testplans/lotus-soup
+            composition_file: testplans/lotus-soup/_compositions/drand-outage-k8s.toml
+          - backend_addr: ci.testground.ipfs.team
+            backend_proto: https
+            plan_directory: testplans/lotus-soup
+            composition_file: testplans/lotus-soup/_compositions/deals-stress-concurrent-natural-k8s.toml
```
Comment (nonsense) [inline .github/workflows/testground-release-issue.yml:73]: @coryschwartz I suggest we have only two tests here:
- `baseline-k8s-3-1.toml` - 1 miner 3 clients
- `paych-stress-k8s.toml`

The rest are not stable, hit various bugs - either in tests or in Lotus, and at the moment should not be part of a release checklist.

File: .github/workflows/testground-release-issue.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  testground-run:
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - backend_addr: ci.testground.ipfs.team
+            backend_proto: https
```
Comment (nonsense) [inline .github/workflows/testground-release-issue.yml:39]: Graphsync testplan should not be part of the Lotus release I think.

**Comment by Stebalien** [comment]
@jennijuju the call is yours here. We already test these in CI, the question is whether or not we want them tracked on the release issue itself. Given that we make a release _pr_, I think that's not really critical. But up to you.

**Comment by jennijuju** [comment]
Will make a decision by Friday.

**Comment by coryschwartz** [comment]
Thanks @jennijuju

**Comment by jennijuju** [comment]
@coryschwartz thanks for the pr - I will close this for now but would definitely revisit this after we have a more stable and automated release process. 
With the process we have today, I think we'll add some overhead with trying to link all pr to a dedicated release issue so we can get the test results there. Also, the way we use release issue now is more for external communication. As we slowly fix all flakey tests and set the expectation to only merge PR and make a release when all tests pass + we dont have a high test ground test coverage atm - this comment in the release issue won't provide too much meaning info.

**Pull request closed without merging**